### DISCUSSION TITLE
January 19, 2022 Community Meeting

### DISCUSSION THREAD

**Discussion by pnbrown** [vmware-tanzu/community-edition#2888] [Community Meeting Notes]
[Recording](https://youtu.be/RFg2mzHTaLc)

- Welcome Back
    - Happy New Year
- Objective
- Announcements
    - [Tanzu to Go in Dallas, Texas on January 20](https://connect.tanzu.vmware.com/VMware-Tanzu-To-Go-Dallas.html)
    - [ADOPTERS.md](https://github.com/vmware-tanzu/community-edition/blob/main/ADOPTERS.md)
    - Community Shoutouts
- Meet the Release Engineering Team!
- PM Updates
    - v0.10.0 release and release candidate
    - Roadmap update
- Engineering Updates
    - Unmanaged Clusters merged into main
    - v0.10.0 Release Update
    - Proposals
        - **2266 [status/completed]**: [Replace tanzu standalone-cluster model with tanzu standalone model](https://github.com/vmware-tanzu/community-edition/issues/2266)
        - **2462 [status/accepted]**: [Add kpack as a tanzu package in the TCE Package Repo](https://github.com/vmware-tanzu/community-edition/issues/2462)
        - **2689 [status/accepted]**: [Add cert injection webhook as package in the TCE package repo](https://github.com/vmware-tanzu/community-edition/issues/2689)
        - **2730 [status/pending(RFC)]**: [Improve visibility into management and workload cluster creation #2730](https://github.com/vmware-tanzu/community-edition/issues/2730)
- Advocacy Updates
    - [Leigh]:
        - Recording posted: [Tanzu Community Edition (plus Doom!) with John McBride](https://www.youtube.com/watch?v=frYvLw8D_80&list=PLAdzTan_eSPTFHMJ1JeRHIrfk__DSfv8f&index=1)
        - Tanzu Tuesday /w Scott yesterday
            - "How to Customize TCE and Add Additional Value Using the Carvel Tool Set"
            - Stream was awesome -- thanks for joining
        - Tanzu Tuesday /w Scott Feb 8
            - "TAP overview and impressions"
            - plan to use TCE underneath the demos
        - Tanzu Tuesday /w Scott Feb 22
            - "Carvel packaging as a unified package manager for Kubernetes"
    - [Robert]:
        - ITQ Conversations /w Robert & Carlos recording a video speaking about provisioning TCE automagically on VMC-on-AWS. Dropping in early Feb. 
- Open Discussion
    - [@PJ] Running a prow job for vulnerability scanning of images in Tanzu packages
        - [Aligning with scanning for k/k](https://github.com/kubernetes/sig-security/tree/main/sig-security-tooling/vulnerability-mgmt) 
        - Previous discussion on [slack](https://kubernetes.slack.com/archives/C02GY94A8KT/p1642104464109300)

**Comment by pj** [comment]
Wrong @pj
